src/everest/config/everest_config.py

```diff
@@ -440,6 +443,14 @@ def validate_maintained_forward_models(self):
                 validate_forward_model_configs(self.forward_model, self.install_jobs)
         return self
 
+    @model_validator(mode="after")
+    def validate_maintained_forward_model_write_objectives(self):
```

Comment:
Should there be a return value here or are these validators different?

Reply:
I am unsure if we should add the return type, but I can add it. Mypy should not complain about it either way because of the decorator I think